doc/src/tutorial/compile_windows_mingw-w64.dox

```diff
@@ -0,0 +1,45 @@
+/** @page compile_windows_mingw-w64 Building PortAudio for Windows with MinGW-w64
+@ingroup tutorial
+
+@section comp_mingw-w64_1 PortAudio for Windows with MinGW-w64
+
+MinGW-w64 is a port of Linux toolchains (gcc, clang) to Windows. It comes packaged with Msys2 in its preferred configuration. Msys2 provides a Linux environment and the package manager pacman.
+Install MinGW-w64 from https://www.msys2.org/. Follow the instructions on this page.
+
+Brief excursion: there are many toolchain variants in Msys2, and you must use the same toolchain to compile all projects together. In this tutorial, we will use the msvcrt-gcc toolchain, simply because it's already present on all Windows versions. Readers may eventually want to switch to the ucrt-gcc toolchain later, for their Windows 10+ builds, although there is hardly any benefit to doing so. A list of available toolchains is at https://www.msys2.org/docs/environments/.
+
+In addition, you have a choice of Windows audio APIs. Here's a summary:
+
+WASAPI: recommended by Microsoft. Supported since Windows Vista (includes 7/8/10+). This should be the default in most cases; it has the best performance of the APIs that work out of the box. If you want to support Windows XP or earlier, you will need to add in another API as a fallback. See the diagram at the top of \ref api_overview for a list of Windows APIs.
```

Comment:
Delete all from "This should be the default..."

Reply:
I deleted all the commentary on WASAPI as requested, but I believe that mentioning the latency advantage of WASAPI would be useful, as this is what a user would want to know.

A brief mention of the option for fallback APIs would help the user know what properties are important when he is evaluating the APIs. It's not specific to Windows APIs, although it is directly relevant to the WASAPI/WMME case.